### ISSUE TITLE
Add possibility to customize focus styles

### ISSUE THREAD

**Issue by vozbrann** [Panenco/panenco-ui#340]

**Comment by lazzystep** [comment]
@vozbrann Hi mate! Provide more details, please. Which component, desired behaviour, etc.

**Comment by vozbrann** [comment]
@lazzystep 
There was a request from customers to remove the black focus outline. Currently, changes should be made in each component. We should consider the possibility of general focus customization.

Currently, inputs have different focus styles. Maybe it should be changed there first?

<img width="277" alt="image" src="https://user-images.githubusercontent.com/39827437/212357512-367e1a1d-89c5-4d18-875b-297f37a86e81.png">
<img width="454" alt="image" src="https://user-images.githubusercontent.com/39827437/212357744-4f9fdace-c5d6-4f9f-bd00-012538915816.png">
<img width="466" alt="image" src="https://user-images.githubusercontent.com/39827437/212357851-cad76b5d-ea4a-437f-80c0-f53fcafb7f9c.png">

**Comment by lazzystep** [comment]
Ok, we should think about best way how to implement it. Either it's a separate theme value or base input which will be reused in other component with it. Should be discussed on some labs meeting
On hold for now
